packages/vue/src/index.js

```diff
@@ -0,0 +1,68 @@
+import Vue from "vue";
+import microlink from "@microlink/vanilla";
+
+const camelCase = string =>
+  string.replace(/-([a-z])/g, g => g[1].toUpperCase());
+
+const createAnchor = url => {
+  const anchor = document.createElement("a");
+  anchor.href = url;
+  anchor.innerHTML = url;
+
+  return anchor;
+};
+
+export const Microlink = Vue.component("Microlink", {
+  props: {
+    url: {
+      type: String,
+      required: true
+    }
+  },
+
+  mounted() {
+    this.renderCard();
+  },
+
+  methods: {
+    renderCard() {
+      const { cardSpace } = this.$refs;
+      const anchor = createAnchor(this.url);
+
+      microlink(anchor, this.parameters, cardSpace);
+    }
+  },
+
+  watch: {
+    $attrs: {
+      handler() {
+        this.renderCard();
+      },
+      deep: true
+    }
+  },
+
+  computed: {
+    parameters() {
+      return Object.keys(this.$attrs).reduce((acc, value) => {
+        if (value.startsWith("data-")) {
+          return acc;
+        }
+
+        return {
+          ...acc,
+          [camelCase(value)]: this.$attrs[value]
+        };
+      }, Vue.microlinkGlobalOptions || {});
+    }
+  },
+
+  template: `<div class="microlink_vue_dom" ref="cardSpace" />`
```

Comment:
@Kikobeats there are some new classNames for us to add to the docs:

* `microlink_vue_dom`: The "base div" that Vue needs to render the React DOM
* `microlink_vanilla_dom`: The "base div" that Vanilla needs to render the React DOM

Reply:
I wrote this into to PR description 👍